plenum/common/stashing_router.py

```diff
@@ -5,12 +5,9 @@
 from sortedcontainers import SortedListWithKey
 
 from common.exceptions import LogicError
+from plenum.server.replica_validator_enums import DISCARD
 from stp_core.common.log import getlogger
 
-DISCARD = -1
-PROCESS = 0
-STASH = 1
-
```

Comment:
I think this should not be dependent on anything replica-related. Actually, replica_validator_enums should depend on it.

Reply:
Yes, but we need to use the same enums in old and new stashers, as now they are not compatible and it will lead to problems at least on integration of Ordering Service step.
I agree that it's better to choose the constants from stashing_router, but it will require changes in existing stashers and validators. 
As I know, this part is being changed in the scope of INDY-2169.

Reply:
I think a simple Actually I don't see how they can be so incompatible. A simple fix would be to keep generic constants in stashing_router, reimport them into old stasher (removing old DISCARD), and base old `STASH_*` constants as new `STASH + n`

Reply:
I think this can be fixed as easy as removing old `DISCARD`, importing new `DISCARD` and `STASH` into old stasher and making old `STASH_*` constants as new `STASH + n`